### DISCUSSION TITLE
Could a maintainer release a new version?

### DISCUSSION THREAD

**Discussion by tomodachi94** [mozilla/rhino#1742] [General]
There have been [241 commits since the last release](https://github.com/mozilla/rhino/compare/Rhino1_7_15_Release...master). 

This is preventing me from [updating Rhino **from 1.7R2** in Nixpkgs](https://github.com/NixOS/nixpkgs/pull/353254), as I need to be able to compile `rhino-all` for some packages like LibreOffice. Unfortunately, the patch that introduces `rhino-all` has not yet been released.

Could the maintainers release a new version? I understand that it could take some time, but I would greatly appreciate it 🙂

**Comment by gbrail** [comment]
Yes! I keep thinking that we'll get a break. I think that the next few
weeks are good. I'm going to open a discussion on the version number.

**Reply by tomodachi94** [reply]
Thank you for the hasty response ❤️
